os/binfmt/binfmt_loadbinary.c

```diff
@@ -194,6 +206,20 @@ int load_binary(int binary_idx, FAR const char *filename, load_attr_t *load_attr
 	heap_table[binary_idx] = bin->sections[BIN_HEAP];
 #endif
 
+#ifdef CONFIG_SYSTEM_DEBUG_ANALYSER_TOOL
```

Comment:
This config is the app config. It looks not proper to use in os. How about using `CONFIG_DEBUG_SYSTEM` or something else which is defined in Kernel?